libs/datasets/combined_datasets.py

```diff
@@ -317,6 +321,10 @@ def datasource_regions(
         CANScraperStateProvidersWithoutFLCounties,
         CANScraperCountyProviders,
     ],
+    CommonFields.VACCINATIONS_ADDITIONAL_DOSE_PCT: [
```

Comment:
Looking back now, I think this could be removed. But, the snapshots I've generated had included this and were successful. I'm not sure if there's a way to test if taking this out would cause problems without another full pipeline run. 

I'm willing to re-run the pipeline without this included to check if we want to try to remove this.

Reply:
Yeah, let's remove it.  You can just check this in and kick off a snapshot and I'll check it before going to bed to make sure it built okay.